.github/workflows/build-and-tests.yml

```diff
@@ -183,6 +183,17 @@ jobs:
               rustup target add riscv64gc-unknown-linux-gnu &&
               npm run build:napi -- --release --target riscv64gc-unknown-linux-gnu &&
               riscv64-linux-gnu-strip *.node
+          - host: ubuntu-latest
+            target: s390x-unknown-linux-gnu
+            setup: |
+              sudo apt-get update
+              sudo apt-get install gcc-s390x-linux-gnu -y
+            build: >-
+              set -e &&
+              CARGO_TARGET_S390X_UNKNOWN_LINUX_GNU_LINKER=s390x-linux-gnu-gcc &&
```

Comment:
Don't you need an "export" here (given that it is in its own bash statement)?

Reply:
Good catch, added it to the line.